test/integration/features/stories/restart.feature

```diff
@@ -0,0 +1,88 @@
+@restart
+Feature: Restart
+Python developer checks out whether OpenShift is something she would like to use for her Django projects.
+It is a first contact, just a swift try...
+
+  Scenario: User starts Minishift using few extra flags
+   Given Minishift has state "Does Not Exist"
+     And image caching is enabled
+    When executing "minishift start --cpus 3 --disk-size 25gb" succeeds
+    Then Minishift should have state "Running"
+
+  Scenario: User access OpenShift console
+   Given with up to "20" retries with wait period of "3" seconds container name "k8s_webconsole_webconsole" should be "running"
+    When executing "minishift console --url" succeeds
+     And stdout is valid URL
+    Then with up to "40" retries with wait period of "500ms" the "status code" of HTTP request to "/console" of OpenShift instance is equal to "200"
+     And "body" of HTTP request to "/console" of OpenShift instance contains "<title>OpenShift Web Console</title>"
+
+  Scenario: User tries oc
+  as she is more like a command line person.
+  (oc environment was set automatically by the test suite here)
+    When executing "oc status" succeeds
+    Then stdout should contain
```

Comment:
@agajdosi I would avoid the stdout content moving forward until it really necessary to compare something, here only exit code of `oc status` would be good enough.